### ISSUE TITLE
How to skip lines starting with a certain char sequence, not just single char

### ISSUE THREAD

**Issue by kurpav00** [jtablesaw/tablesaw#1185]
I have a tab-separated file and I would need to read it into a table while skipping all comment lines starting with '##'. While there seems to be an option to specify a comment prefix as a single character (commentPrefix), I would need to be able to specify a char sequence as a comment prefix. Is this possible? (Some of the data rows start with a single '#', so simply specifying '#' as the comment char is not possible; only lines starting with '##' must be skipped)

**Comment by lwhite1** [comment]
No. there's no support for that. You would have to load the whole table and
delete the ## rows, or do some pre-processing to change the input data.

**Comment by lwhite1** [comment]
I don't think it is implemented anywhere in tablesaw. In the CSV
quasi-standard, a # starting a line is considered a comment marker. IIRC,
this is handled by the CSV library we call.

**Comment by kurpav00** [comment]
OK, thanks. So file preprocessing is probably the only way to go.
